src/Common/src/CoreLib/System/Runtime/InteropServices/IDispatch.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
`src/Common/src/CoreLib` is meant to be used just for the files actually used by CoreLib. Since this is not used by CoreLib, this should be somewhere else. `src\Common\src\Interop\Windows\Ole32\Interop.IStream.cs` is example of prior art.